### PR TITLE
Kang2019decoupling_1_aabayomi

### PR DESCRIPTION
**Pull request by aabayomi**

### CODE REVIEW

File: _summaries/kang2019decoupling_1.md [hunk 1]
```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
...
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+The introduces a two-fold approach to solving long-tail distribution problem in image recognitions by using representation learning and classification during the training process.
+
+### How is it realized (technically)?
+
+Long-tail distribution has been a common problem in real world recognition tasks because of imbalance in data classes.
```
Comment (timchen0618) [inline _summaries/kang2019decoupling_1.md:15]: can you illustrate on how they actually decouple the two (representation learning and classification) more clearly?

File: _summaries/kang2019decoupling_1.md [hunk 2]
```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
...
+- Square-root sampling 
+
+- Progressively-balanced sampling.
+
+
+[2] Classifications 
+
+The authors used this approaches to rebalance the classifier weights without any additional retraining.
+
+- Classifier Re-training (cRT).
```
Comment (timchen0618) [inline _summaries/kang2019decoupling_1.md:34]: they actually do re-training with this...

File: _summaries/kang2019decoupling_1.md [hunk 3]
```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
...
+
+### How is it realized (technically)?
+
+Long-tail distribution has been a common problem in real world recognition tasks because of imbalance in data classes.
+
+[1] Learning representations
+
+During training the authors used different sampling methods.Some of the methods are listed.
+
+- Instance-balanced sampling
```
Comment (timchen0618) [inline _summaries/kang2019decoupling_1.md:21]: could explain each of the method using a sentence? I mean it would be clearer if people haven't read the paper

File: _summaries/kang2019decoupling_1.md [hunk 4]
```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
...
+
+SGD optimizer with momentum 0.9, batch size 512
+cosine learning rate schedule from 0.2 - 0 on  224 × 224 image resolution. 
+
+First representation learning stage, the backbone network is usually trained for 90 epochs. In the second stage, i.e., for retraining a classifier (cRT), we restart the learning rate and train it for 10 epochs while keeping the backbone network fixed.
+
+
+![Sampling-1](kang2019decoupling_1a.png)
+
+Comparison among different sampling methods
```
Comment (timchen0618) [inline _summaries/kang2019decoupling_1.md:68]: I think you can elaborate more on the results, like what the author say about joint or decoupling methods. Which is better? why are they better?

File: _summaries/kang2019decoupling_1.md [hunk 5]
```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
...
+
+![Sampling-2](kang2019decoupling_1b.png)
+
+ResNeXt-50 as backbone was fine-tuned with smaller (0.1×) learning rate on the last block and only retraining the linear classifier  
+
+![Sampling-3](kang2019decoupling_1c.png)
+
+Performance changes for the τ-normalized classifier varies and τ increases from 0 where many-shot accuracy decays dramatically while few-shot accuracy increases.
```
Comment (timchen0618) [inline _summaries/kang2019decoupling_1.md:76]: don't understand this sentence actually...   "Performance changes varies?"

File: _summaries/kang2019decoupling_1.md [hunk 6]
```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
...
+First representation learning stage, the backbone network is usually trained for 90 epochs. In the second stage, i.e., for retraining a classifier (cRT), we restart the learning rate and train it for 10 epochs while keeping the backbone network fixed.
+
+
+![Sampling-1](kang2019decoupling_1a.png)
+
+Comparison among different sampling methods
+
+![Sampling-2](kang2019decoupling_1b.png)
+
+ResNeXt-50 as backbone was fine-tuned with smaller (0.1×) learning rate on the last block and only retraining the linear classifier
```
Comment (timchen0618) [inline _summaries/kang2019decoupling_1.md:72]: same here. what was the author trying to say with this figure?

File: _summaries/kang2019decoupling_1.md [hunk 7]
```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
...
+ResNeXt-50 as backbone was fine-tuned with smaller (0.1×) learning rate on the last block and only retraining the linear classifier  
+
+![Sampling-3](kang2019decoupling_1c.png)
+
+Performance changes for the τ-normalized classifier varies and τ increases from 0 where many-shot accuracy decays dramatically while few-shot accuracy increases.
+
+
+![Sampling-4](kang2019decoupling_1d.png)
+
+Result using ImageNet-LT, comparison between backbones and SOTA methods .
```
Comment (timchen0618) [inline _summaries/kang2019decoupling_1.md:81]: Same here. It would be good if you can include some of the discussion (or conclusions) the author give about the figure, instead of just saying this is the result of XXX and YYY

File: _summaries/kang2019decoupling_1.md [hunk 8]
```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
...
+
+
+<!-- * What interesting variants are explored? -->
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+
+* Long-tail problems can be solves using the right data sampling strategy
```
Comment (timchen0618) [inline _summaries/kang2019decoupling_1.md:92]: can be solved

File: _summaries/kang2019decoupling_1.md [hunk 9]
```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
...
+
+* Instance-balanced sampling  is a more generalizable  approaches  to learning representations.
```
Comment (timchen0618) [inline _summaries/kang2019decoupling_1.md:94]: approaches -> approach

**Comment by timchen0618** [comment]
Hi, can you see my reviews? If so, can you please take a look and consider changing some of your summary?

**Comment by aabayomi** [comment]
@timchen0618  Thank you. I have made the changes.Please let me know if anymore changes are needed.

**Review by zhaoyue-zephyrus — APPROVED** [review]
